Marlin/src/feature/pause.cpp

```diff
@@ -143,13 +143,14 @@ bool pause_print(const float &retract, const float &z_lift, const float &x_pos,
   // Save current position
   stepper.synchronize();
   COPY(resume_position, current_position);
-
+  set_current_to_destination();            // Reset all old destinations to current
   if (retract) {
     // Initial retract before move to filament change position
     set_destination_to_current();
     destination[E_AXIS] += retract;
     RUNPLAN(PAUSE_PARK_RETRACT_FEEDRATE);
     stepper.synchronize();
+    set_destination_to_current(); // apply actual position value after move
```

Comment:
We have a doblockmove just after , and then , we have to set correctly the actual current position that is equal to the destination just before RUNPLAN